frontend/src/view/UserAvatar/UserAvatar.js

```diff
@@ -0,0 +1,8 @@
+import { useState } from 'react';
+import { StyledUserAvatar } from './UserAvatar.styled';
+
+export const UserAvatar = ({ userAvatarImage }) => {
+  const [avatarUrl, setAvatarUrl] = useState(userAvatarImage);
```

Comment:
Ogólnie, jeśli czegoś nie używamy i zostawiamy coś, bo w przyszłości może się przyda, to raczej nie powinniśmy tego zostawiać. Nawet takie memy są o programistach :D